css/main.css

```diff
@@ -0,0 +1,258 @@
+*{
+    padding: 0;
+    margin: 0;
+    font-family: sans-serif;
+    font-weight: 100;
+    box-sizing: border-box;
```

Comment:
Research on and specify vendor prefixes when using box-sizing, to make it compatible across all browsers.

Reply:
Solved!